twoliter/embedded/metadata.spec

```diff
@@ -0,0 +1,64 @@
+%global cross_generate_attribution %{nil}
```

Comment:
I didn't want licenses for the metadata package to show up on the image - it's kind of an implementation quirk - which is why this is set to `%{nil}`.

I thought about force-erasing the RPM so it didn't show up in the generated application-inventory.json, or filtering it out, but that seemed less important and a bit too "magical".